starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -51,8 +51,8 @@ public List<Inventory> findAll() {
    * @return Created/Updated Inventory.
    */
   public Inventory create(Inventory inventory) {
-    // TODO
-    return null;
+    inventory.setId(null);
+    return this.mongoTemplate.insert(inventory);
```

Comment:
Is there a way we can create and/or update an existing inventory object with a single method call, something besides insert()?